internal/controller/components/kueue/kueue_controller.go

```diff
@@ -57,14 +56,7 @@ func (s *componentHandler) NewComponentReconciler(ctx context.Context, mgr ctrl.
 	b := reconciler.ReconcilerFor(mgr, &componentApi.Kueue{})
 
 	if cluster.GetClusterInfo().Version.GTE(semver.MustParse("4.17.0")) {
-		// "own" VAP, because we want it has owner so when kueue is removed it gets cleaned.
-		b = b.OwnsGVK(gvk.ValidatingAdmissionPolicy)
-
-		// "watch" VAPB, because we want it to be configurable by user, and it can be left behind
-		// when kueue is removed
-		b = b.WatchesGVK(gvk.ValidatingAdmissionPolicyBinding)
-
-		// add OCP 4.17.0 specific menifests
+		// add OCP 4.17.0 specific manifests
```

Comment:
but we still wanna call an emptry kustomization.yaml https://github.com/opendatahub-io/opendatahub-operator/blob/e6f0acebfd8c4d825b5afeb1c3567c3238f320e9/config/kueue-configs/ocp-4.17-addons/kustomization.yaml ?

Reply:
do it